NeuroID/src/main/java/com/neuroid/tracker/service/NIDServiceTracker.kt

```diff
@@ -81,6 +81,7 @@ object NIDServiceTracker {
             put("userId", sharedDefaults.getUserId())
             put("clientId", sharedDefaults.getClientId())
             put("identityId", sharedDefaults.getUserId())
+            put("registeredUserID", sharedDefaults.getRegisteredUserId())
```

Comment:
ID needs to be Id here
https://github.com/Neuro-ID/trackjs/blob/main/src/collection/event-queue.js#L170

Reply:
fixed. thanks!